src/pyscipopt/scip.pxi

```diff
@@ -5704,13 +9225,21 @@ cdef class Model:
         return score
 
     def getTreesizeEstimation(self):
-        """Get an estimation of the final tree size """
+        """
+        Get an estimation of the final tree size.
+
+        Returns
+        -------
+        int
```

Comment:
```suggestion
        float
```

Reply:
Is this actually true? I get that the function returns SCIP_Real, but isn't it always an integer?

Reply:
I'm not sure @Opt-Mucca. I don't know how the estimate is being done (English-wise, shouldn't it be `getTreeSizeEstimate`?), but it wouldn't surprise me if it allowed non-integer values.